__tests__/components/Footer.test.tsx

```diff
@@ -1,6 +1,6 @@
 import { renderWithLanguage } from '../../__helpers__'
 
-import { staticPageRoutes, SPANISH } from '../../constants'
+import { SPANISH, PrivacyPolicyRoute } from '../../constants'
 import Footer, { copyright, linkInfo } from '../../components/Footer'
 
 describe('<Footer />', () => {
```

Comment:
@vsauceda-cfa  as a general thing, it is better to add new tests instead of changing old ones to help prevent code regression. As I'm looking at this though, that looks like it may not have been possible because of how I wrote the tests. Is that correct?

Reply:
Yea. I am very new to test. Havent done much in the past so I was just reverse engineering and trying to get it to work. I was able to figure out how it was working

Reply:
@vsauceda-cfa do you think you could make it work without changing the old test?